client-ci-analysis/ci-testfailure-analyzer/readme.md

```diff
@@ -0,0 +1,7 @@
+﻿# ci-testfailure-analyzer
+
+Steps to use:
+
+1. Go to AzDo, view build, view and download build logs for run test phase
```

Comment:
If this tools looks at the historical logs why do we need a specific build logs from the run test phase?
Does this apply to any phase?

Reply:
@nkolev92 
I corrected `readme.md` file.